### DISCUSSION TITLE
bangua (iTaigi-like) 产品定位讨论

### DISCUSSION THREAD

**Discussion by Guanchishan** [MindongLab/yngdieng#850] [产品功能 ideas]
春天花花@微信：

> 主要是讓用戶（市民）可以參與進來，自己上傳語音+解釋+地區，保證語料來源的豐富性和多樣性。我知道現在平話中使用者最多的是福州話，參與保護的人也比較多，但其他地區的平話其實也非常類似，只是讀音有一些變化

@Guanchishan :
> 榕典是多来源平话词典，为了补传统词汇之不足有设用户贡献区，但基本上是收录受到承认、已存在于平话中的词汇，还要经过验证才能上线
但你提及爱台语，这个网站他会允许群力群策创造新词汇，我觉得这就是和榕典不一样的地方了，不过很有意思，很值得做，有活化语言的意义
我对爱台语的理解是，这个网站不提供一种教学与规范，而是提供了一个切磋琢磨、投票甄选的平台

暂时讨论结果是，可以作为榕典内的一个模块，该模块向着iTaigi的方向发展。

模块的意义：现在因为推普，自然的语言交流被打断，如果能靠这个平台重新发生新词是好事 -- @Guanchishan

**Comment by Guanchishan** [comment]
@ztl8702  我前面就想講，汝講的「說咱閩南語-like」產品很合適放這個功能。

**Reply by ztl8702** [reply]
![image](https://user-images.githubusercontent.com/2714446/106821985-30e18100-66b9-11eb-865d-e63554be4013.jpeg) ![image](https://user-images.githubusercontent.com/2714446/106821996-363ecb80-66b9-11eb-9d18-a31c510ac26e.png)
